WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -406,20 +384,11 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
 
     override fun onStorySaveButtonPressed() {
         analyticsTrackerWrapper.track(PREPUBLISHING_BOTTOM_SHEET_OPENED)
-        showPrepublishingBottomSheet()
-    }
-
-    private fun setUntitledStoryTitleIfTitleEmpty() {
-        if (editPostRepository.title.isEmpty()) {
-            val untitledStoryTitle = resources.getString(R.string.untitled)
-            storyRepositoryWrapper.setCurrentStoryTitle(untitledStoryTitle)
-            updateStoryPostTitleUseCase.updateStoryTitle(untitledStoryTitle, editPostRepository)
-        }
+        viewModel.openPrepublishingBottomSheet()
```

Comment:
We might want to consider renaming the method to "viewModel.onStorySaveButtonPressed" and moving both "analyticsTrackerWrapper.track(PREPUBLISHING_BOTTOM_SHEET_OPENED)" and "openPrepublishingBottomSheet()" there. The view layer ideally shouldn't know/care what is going to happen on "buttonPress". Wdyt?

Reply:
Resolved here https://github.com/wordpress-mobile/WordPress-Android/pull/12501/commits/c61b51fd52bc98dc2e35569afdd91e9e711f57d0